tests/nc-client/main.c

```diff
@@ -101,6 +103,17 @@ callback_connected(void * cookie, int socket)
 	/* Record socket for future use. */
 	send->socket = socket;
 
+	/* Make socket blocking. */
+	if ((opts = fcntl(send->socket, F_GETFL, 0)) == -1) {
+		warnp("fcntl get");
+		goto err0;
+	}
+	opts = opts & (~O_NONBLOCK);
+	if (fcntl(send->socket, F_SETFL, opts) == -1) {
+		warnp("fcntl set");
+		goto err0;
+	}
+
```

Comment:
Hmm, I added that while I was trying to figure out the latest set of test failures, and it didn't hurt anything.  I've removed it now.